### DISCUSSION TITLE
Option to switch Climate off when no motion.

### DISCUSSION THREAD

**Discussion by fierland** [jmcollin78/versatile_thermostat#486] [Ideas]
Is it possible to add the option to turn the thermostat off at no motion. Now you only can select presets, but for an arco in a room i just want to turn it off when nobody is in there.

**Comment by jmcollin78** [comment]
Hello @fierland ,

As explained many times, Vtherm will never turn a device on or off (unless for window detection). I consider turning on and off is the choice of the user, and I'm sure you don't want that your heater turns on automatically when you are not there for a long time (for example).

For over_climate VTherm like AC device, ma ny users claims for that function. So I will certainly add this in the future.

So you can do, is to create an automation while waiting for implementation.
